Sparkle/SUSystemUpdateInfo.m

```diff
@@ -59,9 +59,27 @@ + (BOOL)systemAllowsAutomaticUpdatesForHost:(SUHost *)host
 }
 
 #define NUM_UPDATE_GROUPS 7
-+ (NSUInteger)updateGroupForHost:(SUHost *)host {
-    NSNumber* updateGroupIdentifier = [host updateGroupIdentifier];
++ (NSUInteger)updateGroup {
+    NSNumber* updateGroupIdentifier = [self updateGroupIdentifier];
     return ([updateGroupIdentifier unsignedIntValue] % NUM_UPDATE_GROUPS);
 }
 
++ (NSNumber*)updateGroupIdentifier {
+    NSNumber* updateGroupIdentifier = [[NSUserDefaults standardUserDefaults] objectForKey:SUUpdateGroupIdentifierKey];
```

Comment:
Please use hour wrapper around defaults, because we can't always use standard defaults.

Reply:
Done. Please check again as I refactored some class methods to instance methods